fcrepo-http-commons/src/test/java/org/fcrepo/http/commons/domain/PreferTagTest.java

```diff
@@ -32,4 +35,15 @@ public void testEmpty() {
         assertEquals("", preferTag.getValue());
         assertTrue(preferTag.getParams().isEmpty());
     }
+
```

Comment:
It looks like you may have done a force-push. In the future, please add subsequent commits to the same branch instead of mashing updates into a single commit. That way it is easier to review and see your follow-on updates.